index.js

```diff
@@ -146,10 +189,12 @@ function parse (opts) {
   const extensions = opts.extensions
 
   const deps = {}
-  const paths = []
   const seen = []
   const core = []
```

Comment:
Can we move to sets and maps now we're on a high enough node.js version?

Reply:
We can do this post `4.0.0`?